egi_notebooks_hub/d4science.py

```diff
@@ -188,11 +188,12 @@ async def get_resources(self, access_token):
     async def authenticate(self, handler, data=None):
         # first get authorized upstream
         user_data = await super().authenticate(handler, data)
-        context = quote_plus(getattr(self, "d4science_context", None))
+        context = getattr(self, "d4science_context", None)
```

Comment:
i guess this will need to be updated when this is merged https://github.com/EGI-Federation/egi-notebooks-hub/pull/117